client/js/shout.js

```diff
@@ -314,6 +314,11 @@ $(function() {
 		}
 	});
 
+	socket.on("topic", function(data) {
+		// .text() escapes HTML (but not quotes)
```

Comment:
@kalaksi I just tested `/topic I'm a "test"` and it works just fine. Probably best if you could amend your commit and remove the parenthesis (but keep the comment on `.text()`, it's useful and relevant here).